app/adapters/goldencobra/article_adapter.rb

```diff
@@ -0,0 +1,66 @@
+require "addressable/uri"
+
+module Goldencobra
+  class ArticleAdapter
+    class << self
+
+      # Alias method for find_by_url
+      # @param url [String] "http://www.IkusEI.de/asdf/?asAAAa=12"
+      # @param options = {} Options of method find_by_url
+      #
+      # @return [Integer] ArticleID
+      def find(url, options = {})
```

Comment:
shorter call `Goldencobra::ArticleAdapter.find("http://www...")` which would by used most of the time like `User.find(2)` and `User.find_by_id(2)`